src/test/java/hudson/plugins/copyartifact/DownstreamBuildSelectorTest.java

```diff
@@ -31,7 +31,7 @@
 import java.util.Map;
 import java.util.Set;
 
-import hudson.plugins.copyartifact.testutils.CopyArtifactUtil;
+import hudson.plugins.copyartifact.testutils.CopyArtifactUtil;

```

Comment:
Note: This changes the line break from LF to CR+LF.
The root cause is inconsistent line breaks introduced in #55.
(I couldn't find them in the review with github web interfaces...)

I'll accept this change even though pull requests would better not to contain unrelated changes, as:
* Not every developers use editors that can handle files with inconsistent line breaks.
* This should be fixed anyway.